ios/Dora/Images.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Is this just generated?

Reply:
It sometimes is, but in this case, I had to copy the list from online to make it use the assets I wanted.